pkg/apis/kubermatic/v1/configuration.go

```diff
@@ -42,6 +42,24 @@ const (
 // OperationType is the type defining the operations triggering the compatibility check (CREATE or UPDATE).
 type OperationType string
 
+// +kubebuilder:validation:Enum=Addon;Cluster;ClusterBackup;Project;None
+// MetricsCollector is the name of an available metrics collector.
+type MetricsCollector string
+
+const (
+	// AddonCollector is addon metrics collector.
+	AddonCollector MetricsCollector = "Addon"
+	// ClusterBackupCollector is cluster backup metrics collector.
+	ClusterBackupCollector MetricsCollector = "ClusterBackup"
+	// ClusterCollector is cluster metrics collector.
+	ClusterCollector MetricsCollector = "Cluster"
+	// ProjectCollector is project metrics collector.
+	ProjectCollector MetricsCollector = "Project"
+	// NoneCollector is a special name that points to no collector.
+	// It is used when a client wants to disable certain collectors in all seeds except for a few.
+	NoneCollector MetricsCollector = "None"
```

Comment:
I'm not sure I understand the purpose of this special collector. Is it even used anywhere? I couldn't find a reference.

Reply:
For example, consider that I have multiple seeds and I want to disable certain collectors in all of them, except for one seed where all collectors should remain enabled. To achieve this, I can modify the global configuration to disable the specified collectors. However, to enable all collectors in the one specific seed, I need to set an empty list for disabled collectors. The issue is that an empty list means the global configuration applies, which would disable some collectors. To address this, a special value has been added to explicitly indicate that no collectors should be disabled for that specific seed.

Reply:
Ahh. That makes sense. So the value is not used anywhere because it's a stand-in that makes the list non-empty. Clever. Thanks for the explanation!